tgui/packages/tgui/interfaces/Orbit.js

```diff
@@ -0,0 +1,181 @@
+  const { color, orbit, thing } = props;
+
+  return (<Button
+    color={color}
+    content={(
+      <span>
+        {thing.name}{" "}
+        {
+          thing.orbiters
+            ? (
+              <span>
+                ({thing.orbiters}{" "}
+                <img
+                  src="ghost.png"
+                  style={{
+                    opacity: "0.7",
+                  }}
+                />)
+              </span>
+            )
+            : undefined
+        }
+      </span>
+    )}
+    onClick={() => orbit(thing.name)}
+  />);
+};
+
+export const Orbit = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    alive,
+    antagonists,
+    dead,
+    ghosts,
+    misc,
+    npcs,
+  } = data;
+
+  const [search, setSearch] = useLocalState(context, "search", "");
+
+  const collatedAntagonists = {};
+  for (const antagonist of antagonists) {
+    if (collatedAntagonists[antagonist.antag] === undefined) {
+      collatedAntagonists[antagonist.antag] = [];
+    }
+
+    collatedAntagonists[antagonist.antag].push(antagonist);
+  }
+
+  const orbit = name => act("orbit", { name });
+
+  return (
+    <Window>
+      <Window.Content>
+        <Section title={
+          <Input
+            fluid
+            value={search}
+            onInput={(_, value) => setSearch(value)}
+          />
+        } />
+
+        { antagonists.length > 0 && (
+          <Section title="Antagonists">
+            { Object.entries(collatedAntagonists).sort((a, b) => {
+              return a.localeCompare(b);
+            }).map(([name, antags]) => {
+              return (
+                <Section key={name} title={name} level={2}>
+                  {
+                    antags
+                      .filter(searchFor(search))
+                      .sort(sortByName)
+                      .map(antag => (
+                        <OrbitedButton
+                          key={antag.name}
+                          color="bad"
+                          thing={antag}
+                          orbit={orbit}
+                        />
+                      ))
+                  }
+                </Section>
+              );
+            }) }
+          </Section>
+        ) }
+
+        <Section title="Alive">
+          { alive.filter(searchFor(search)).sort(sortByName).map(thing => (
+            <OrbitedButton
+              key={thing.name}
+              color="good"
+              thing={thing}
+              orbit={orbit}
+            />)) }
+        </Section>
+
+        <BasicSection
+          title="Dead"
+          source={dead}
+          orbit={orbit}
+          search={search}
+        />
+
+        <BasicSection
+          title="NPCs"
+          source={npcs}
+          orbit={orbit}
+          search={search}
+        />
+
+        <BasicSection
+          title="Misc"
+          source={misc}
+          orbit={orbit}
+          search={search}
+        />
+
+        <BasicSection
```

Comment:
Small suggestion, but could you bump the ghost selection up below "Alive"? I feel like it's used more than NPC's and misc, as people tend to want to see if someone's observing / follow someone talking in ghostchat, etc. Not a huge deal either way tho

Reply:
Sure.